docs/machine-learning/ml-feasibility-study.md

```diff
@@ -1,10 +1,10 @@
 # Feasibility Studies
 
 The main goal of feasibility studies is to assess whether it is feasible to solve the problem satisfactorily using ML with the available data. We want to avoid investing too much in the solution before we have:
-*	Sufficient evidence that a solution would be the best technical solution given the business case
```

Comment:
GENERAL RECOMMENDATION: remove files having changes due exclusively to auto-formatting/whitespace edits from this PR and submit as part of a separate PR (focused on misc. whitespace/formatting updates) in order to reduce the 'noise' in this PR.